src/160._Intersection_of_Two_Linked_Lists.md

```diff
@@ -0,0 +1,95 @@
+## Intersection of Two Linked Lists
+
+### Description
+
+```Python
+Write a program to find the node at which the intersection of two singly linked lists begins.
+
+For example, the following two linked lists:
+```
+
+![](https://myblogs-photos-1256941622.cos.ap-chengdu.myqcloud.com/Leetcode/%E9%93%BE%E8%A1%A8%E4%BA%A4%E9%9B%86_01.png)
+
+**Example 1:**
+
+![](https://myblogs-photos-1256941622.cos.ap-chengdu.myqcloud.com/Leetcode/%E9%93%BE%E8%A1%A8%E4%BA%A4%E9%9B%86_02.png)
```

Comment:
I think this picture should be move into this repo.
otherwise, if the url is broken, we could not show this picture any more.

You could make a dir `resources` in this repo as this directory tree:
|----src
|----resources